# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1303753869 25200
#      Mon Apr 25 10:51:09 2011 -0700
# Branch stable
# Node ID 65faec78a5b37d0dfb037eafe005b7e70a93806d
# Parent  ccfe03bebeba7d3e4dda73020f17ae5acfb1b65d
maint: Don't install main-rcfile, local-rcfile but do distribute them in tarball.

startup/module.mk: Don't install rcfiles.  They are installed by special
Makefile rules which rename them to octaverc and place them in the correct
directory.

diff --git a/scripts/startup/module.mk b/scripts/startup/module.mk
--- a/scripts/startup/module.mk
+++ b/scripts/startup/module.mk
@@ -1,27 +1,25 @@
 FCN_FILE_DIRS += startup
 
 startup_FCN_FILES = \
   startup/__finish__.m
 
+LOCAL_STARTUP_FILE_SRC  = startup/local-rcfile
+
 SYSTEM_STARTUP_FILE_SRC = startup/main-rcfile
 
 SYSTEM_INPUTRC_FILE_SRC = startup/inputrc
 
-LOCAL_STARTUP_FILE_SRC = startup/local-rcfile
-
 STARTUP_FILE_SRC = \
+  $(LOCAL_STARTUP_FILE_SRC) \
   $(SYSTEM_STARTUP_FILE_SRC) \
-  $(SYSTEM_INPUTRC_FILE_SRC) \
-  $(LOCAL_STARTUP_FILE_SRC)
+  $(SYSTEM_INPUTRC_FILE_SRC)
 
 FCN_FILES += \
-  $(startup_FCN_FILES) \
-  $(SYSTEM_STARTUP_FILE_SRC) \
-  $(LOCAL_STARTUP_FILE_SRC)
+  $(startup_FCN_FILES)
 
 EXTRA_DIST += \
-  $(SYSTEM_INPUTRC_FILE_SRC)
+  $(STARTUP_FILE_SRC)
 
 PKG_ADD_FILES += startup/PKG_ADD
 
 DIRSTAMP_FILES += startup/$(octave_dirstamp)
